src/Mesh/babylon.abstractMesh.ts

```diff
@@ -763,7 +763,7 @@
             var yaw = -Math.atan2(dv.z, dv.x) - Math.PI / 2;
             var len = Math.sqrt(dv.x * dv.x + dv.z * dv.z);
             var pitch = Math.atan2(dv.y, len);
-            Quaternion.RotationYawPitchRollToRef(yaw + yawCor, pitch + pitchCor, rollCor, this.rotationQuaternion);
+            this.rotationQuaternion = Quaternion.RotationYawPitchRollToRef(yaw + yawCor, pitch + pitchCor, rollCor, this.rotationQuaternion || new Quaternion());

```

Comment:
Hey... that's totally new :) not even updated on my local repository yet.
This should work then, apologies.